apps/system/style/chrome/chrome.css

```diff
@@ -386,6 +392,8 @@
 .controls .back-button[disabled],
 .controls .forward-button[disabled] {
   display: none;
+  opacity: 0;
+  animation: fadeOut .2s ease-in;
```

Comment:
I don't think it should matter, but if we only have a start an end state, I think a transition would be better. Would prefer to switch to using a transition.